redisdb/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG - redisdb
 
+## 4.7.1
+
+* [Added] fix the length of key of type stream [#14722](https://github.com/DataDog/integrations-core/pull/14722).
+
```

Comment:
```suggestion
```

The changelog gets automatically generated when we release the integration, no need to update it manually 🙂